File: Codingbat/Java/Logic-1/caughtSpeeding.java
public int caughtSpeeding(int speed, boolean isBirthday) {
  if(isBirthday && speed >65){
    if(speed <= 86){
      return 1;
    } else {
      return 2;
    }
  } else if(!isBirthday && speed > 60){
    if(speed >80) return 2;
    else {
      return 1;
    }
  }else {
    return 0;
  }
  
}

File: Codingbat/Python/Logic-1/caught_speeding.py
def caught_speeding(speed, is_birthday):
  if is_birthday: 
    if speed > 85: 
      return 2 
    elif speed > 65 and speed <= 85 : 
      return 1 
    else: 
      return 0
  elif not is_birthday: 
    if speed > 80: 
      return 2 
    if speed > 60 and speed <= 80: 
      return 1 
    else: 
      return 0 
